multiqc/modules/isoseq/isoseq.py

```diff
@@ -0,0 +1,231 @@
+                "name": "11 to 100 CCS",
+            },
+            ">100": {
+                "name": "More than 100 CCS",
+            },
+        }
+
+        self.add_section(
+            name="Cluster size distribution",
+            anchor="isoseq-cluster-size-distribution",
+            description="A distribution of cluster size (number of CC clustered to form one Hifi read).",
+            helptext="""
+            The csv file produced by Iso-Seq cluster shows which CCS have been clustered together to form 
+            one Hifi reads. The bargraph represent the distribution of the cluster size using four 
+            categories : -2, 3-10, 11-100, >100.
+            """,
+            plot=bargraph.plot(
+                plot_data,
+                cats,
+                {
+                    "id": "isoseq-cluster-size-distribution-barplot",
+                    "title": "Iso-Seq cluster: Histogram of cluster size.",
+                    "ylab": "Count",  # Y axis label
+                },
+            ),
+        )
+
+    def _add_general_stats_refine(self, data_by_sample):
+        headers = {
+            "num_reads_fl": {
+                "title": "Full-length",
+                "description": "Number of CCS where both primers have been detected",
+                "scale": "GnBu",
+                "format": "{:,.d}",
+            },
+            "num_reads_flnc": {
+                "title": "Non-chimeric full-length",
+                "description": "Number of non-chimeric CCS where both primers have been detected",
+                "scale": "RdYlGn",
+                "format": "{:,.d}",
+            },
+            "num_reads_flnc_polya": {
+                "title": "Poly(A) free non-chimeric full-length",
+                "description": "Number of non-chimeric CCS where both primers have been detected and the poly(A) tail has been removed",
+                "scale": "GnBu",
+                "format": "{:,.d}",
+            },
+        }
+        self.general_stats_addcols(data_by_sample, headers, namespace="refine")
+
+    def _add_refine_plot(self, values_by_metric_by_sample):
+        data_labels = {
+            "fivelen": "5' primer length",
+            "threelen": "3' primer length",
+            "insertlen": "Insert length",
+            "polyAlen": "Poly(A) length",
+        }
+        data_by_sample_by_metric = {k: {} for k in data_labels.keys()}
+        for sname, values_by_metric in values_by_metric_by_sample.items():
+            for metric in data_labels.keys():
+                data_by_sample_by_metric[metric][sname] = values_by_metric[metric]
+
+        data_labels = [
+            {
+                "label": title,
+                "title": "Iso-Seq refine: " + title,
+            }
+            for metric, title in data_labels.items()
+        ]
+
+        self.add_section(
+            name="Iso-Seq refine",
+            anchor="insert-refine-stats",
+            description="Iso-Seq refine statistics",
```

Comment:
Would be nice if we could have a better description of specifically what `isoseq refine` is doing. PacBio docs don't seem to be very descriptive of this.